src/main/java/uk/ac/ebi/embl/gff3tools/validation/builtin/LengthValidation.java

```diff
@@ -27,29 +29,19 @@ public class LengthValidation implements FeatureValidation {
     private static final String INVALID_INTRON_LENGTH_MESSAGE = "Intron feature length is invalid for accession \"%s\"";
     private static final String INVALID_EXON_LENGTH_MESSAGE = "Exon feature length is invalid for accession \"%s\"";
 
-    @Override
-    public String getValidationRule() {
-        return INTRON_LENGTH_VALIDATION_RULE;
-    }
-
-    @Override
+    @ValidationMethod(type = ValidationType.FEATURE)
```

Comment:
This needs to be splited and the validations must be named. If we are to do this on a different PR a ticket to track progress should be added.